error/runtime_spec.go

```diff
@@ -41,6 +41,73 @@ const (
 	// ReadonlyOnWindows represents the error code of readonly setting test on Windows
 	ReadonlyOnWindows
 
+	// MountInOrder represents the error code of order test of mount list
+	MountInOrder
+	// MountDestinationAbs represents the error code of absolute path test of mount destination
+	MountDestinationAbs
+	// MountNotNestedOnWindows represents the error code of nested mount point on Windows
+	MountNotNestedOnWindows
+	// MountROSupportOnWindows represents the error code of RO test on Windows
+	MountROSupportOnWindows
+
+	// Process
+	// ProcessOnStart represents the error code of process property test when 'start' is called
+	ProcessOnStart
+	// ProcessIngnoreConsoleSize represents the error code of console size test when terminal is `false` or unset
+	ProcessIgnoreConsoleSize
+	// ProcessCWDAbs represents the error code of absolute path test of process `cwd`
+	ProcessCWDAbs
+
+	// PosixProcess
+	// RlimitType represents the error code of rlimit type test
+	RlimitType
+	// RlimitSoftCur represents the error code of rlimit soft test
+	RlimitSoftCur
+	// RlimitHardMax represents the error code of rlimit hard test
+	RlimitHardMax
+	// RlimitDuplicated represents the error code of rlimit duplicate type test
+	RlimitDuplicated
+
+	// LinuxProcess
+	// LinuxCapabilityMap represents the error code of capability map test
+	LinuxCapabilityMap
+	// LinuxOomScoreAdj represents the error code of `oom_score_adj` value test when it is set
+	LinuxOomScoreAdj
+	// LinuxNoOomScoreAdj represents the error code of `oom_score_adj` value test when it is not set
+	LinuxNoOomScoreAdj
+
+	// Platform-specific configuration
+	// WindowsPlatform represents the error code of windows config test on Windows platform
+	WindowsPlatform
+
+	// POSIX-platform Hooks
+	// PosixHookPathAbs represents the error code of absolute path test of posix hook path
+	PosixHookPathAbs
+	// PosixHookTimeout represents the error code of hook timeout test
+	PosixHookTimeout
+	// PosixHookCalledInOrder represents the error code of order test of called hooks
+	PosixHookCalledInOrder
+	// PosixHookStateOverStdin represents the error code of `state` information test if it was passed to hook process
+	PosixHookStateOverStdin
+	// PosixHookPrestartOrder represents the error code of order test of pre-start hook
+	PosixHookPrestartOrder
+	// PosixHookPoststartOrder represents the error code of order test of post-start hook
+	PosixHookPoststartOrder
+	// PosixHookPoststopOrder represents the error code of order test of post-stop hook
+	PosixHookPoststopOrder
+
+	// Annotations
+	// AnnotationKeyValue represents the error code of annotations key-value map test
+	AnnotationKeyValue
```

Comment:
There are [several RFC 2119 conditions for annotations][1], but you only have one code here.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/config.md#annotations

Reply:
I saw they were quoted, should we remove the this quote from the spec?
```
Keys MUST be strings.
Keys MUST NOT be an empty string.
Keys SHOULD be named using a reverse domain notation - e.g. `com.example.myKey`.
Keys using the `org.opencontainers` namespace are reserved and MUST NOT be used by subsequent specifications.
Implementations that are reading/processing this configuration file MUST NOT generate an error if they encounter an unknown annotation key.

Values MUST be strings.
Values MAY be an empty string.
```

Reply:
> I saw they were quoted…

I see some backticks in the spec you reference, but no quotes.  And backticks are just markup, they don't affect the number of RFC 2119 conditions contained in the text.